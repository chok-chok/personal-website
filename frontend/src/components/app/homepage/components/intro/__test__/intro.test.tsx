import * as React from 'react';
import Intro from '../intro';
import { shallow } from 'enzyme';

describe('Test Intro component', () => {
  it('should renders correctly', () => {
    const tree = shallow(<Intro />);
    expect(tree).toMatchSnapshot();
  });
});
