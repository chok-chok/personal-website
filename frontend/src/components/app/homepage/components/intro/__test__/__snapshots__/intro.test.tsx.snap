// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Intro component should renders correctly 1`] = `
<Fragment>
  <styled__IntroImageWrapper>
    <styled__IntroImage
      src="/img/profile.jpg"
    />
  </styled__IntroImageWrapper>
  <styled__IntroHeaderText>
    Product Manager & Programmer
  </styled__IntroHeaderText>
  <styled__IntroBodyText>
    Product manager during the day, software enginner in the evening
  </styled__IntroBodyText>
</Fragment>
`;
